data/data/bootstrap/ovirt/files/usr/local/bin/fletcher8

```diff
@@ -0,0 +1,16 @@
+#!/usr/libexec/platform-python
+
+# Script that uses the fletcher8 algorithm to generate a hash from an input
```

Comment:
1. Where is this script by used?
2. Why is external script used and not code the same in Go in the code where its used?

Reply:
Both script are not needed any more.